src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -558,6 +558,147 @@ def test_aks_create_and_update_with_vpa(
             ],
         )
 
+    @AllowLargeResponse()
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=85258020&view=results) to validate the change.

Reply:
Queued another [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=85407298&view=results) as there are some code changes since last reviewed.

Reply:
Test passed!